inc/customizer/controls/react/src/icon-color/IconColor.js

```diff
@@ -0,0 +1,16 @@
+import ColorControl from '../common/ColorControl';
+
+const IconColor = ({ label }) => {
+	return (
+		<div className="nv-repeater--field">
+			<ColorControl
+				defaultValue=""
+				label={label}
+				onChange={function noRefCheck() {}}
```

Comment:
`noRefCheck()` is a placeholder shown inside the storybook docs, but it actually should be your onChange handler. 

You can simply use the `ColorControl`, instead of wrapping it inside another component.